5-Ansible_Deep_Dive/roles/user/handlers/main.yml

```diff
@@ -0,0 +1,33 @@
+# handlers file for roles/user
+
+- name: Add new user to sudoers
+  lineinfile:
+        path: /etc/sudoers
+        line: "{{ new_user }} ALL=(ALL) NOPASSWD:ALL"
+        state: present
+
+- name: Set SSH
```

Comment:
The `sshd` daemon is not responsible for user ssh keys, so it should not be started or restarted after putting the ssh keys to the remote host.